service/controller/chartconfig/v7/version_bundle.go

```diff
@@ -9,7 +9,7 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "chart-operator",
-				Description: "checking cordon annotations in chart resource to avoid any further update.",
+				Description: "Check cordon annotations in chart resource to avoid any further update.",
```

Comment:
Discussion to change the changelog, https://github.com/giantswarm/chart-operator/pull/250#discussion_r299654871